src/main/java/mate/academy/webshop/dao/jdbc/ProductDaoJdbcImpl.java

```diff
@@ -0,0 +1,127 @@
+package mate.academy.webshop.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.webshop.dao.ProductDao;
+import mate.academy.webshop.lib.Dao;
+import mate.academy.webshop.model.Product;
+import mate.academy.webshop.util.ConnectionUtil;
+import org.apache.log4j.Logger;
+
+@Dao
+public class ProductDaoJdbcImpl implements ProductDao {
+    private static final Logger LOGGER = Logger.getLogger(ProductDaoJdbcImpl.class);
+
+    @Override
+    public Product create(Product product) {
+        String query = "INSERT INTO products (name, price) VALUES (?, ?)";
+        Product createdProduct = null;
+
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(query);
+            statement.setString(1, product.getName());
+            statement.setDouble(2, product.getPrice());
+            statement.executeUpdate();
+            List<Product> list = getAll();
+            createdProduct = list.get(list.size() - 1);
+        } catch (SQLException e) {
+            LOGGER.error("Can't create product", e);
+            return null;
```

Comment:
maybe it's better to throw an exception? return null is not a very good practice

Reply:
That's right. Fixed.